### PR TITLE
Feature/ui switch between bottompanel tabs

### PR DESCRIPTION
**Pull request by ajainarayanan**
- Enables the ability to export a config from studio view.
- Removes commented code.
- Deletes hydrator services no longer used.

TODO:
- The current node selection is broken and it will be fixed in the subsequent PR.
- Still have ```MyAppDAGService``` and a bunch of services to remove. Should be able to prune them in subsequent PRs.

### CODE REVIEW

File: cdap-ui/app/features/hydrator/controllers/create/toppanel-ctrl.js [hunk 1]
```diff
@@ -87,38 +87,25 @@ class TopPanelController{
...
-          //       // Clicking on the hidden download button. #hack.
-          //       $timeout(function() {
-          //         document.getElementById('pipeline-export-config-link').click();
-          //       });
-          //     }.bind(this),
-          //     function error() {
-          //       console.log('ERROR: Exporting ' + MyAppDAGService.metadata.name + ' failed.');
-          //     }
-          //   );
+          var content = exportConf;
```
Comment (elfenheart) [inline cdap-ui/app/features/hydrator/controllers/create/toppanel-ctrl.js]: this is kind of replicating exportConfig right? Do we need to assign it to a different variable?

**Comment by elfenheart** [comment]
small comment, otherwise :+1: LGTM

File: cdap-ui/app/features/hydrator/controllers/create/toppanel-ctrl.js (hunk 1 above)
Comment (ajainarayanan) [inline cdap-ui/app/features/hydrator/controllers/create/toppanel-ctrl.js]: So this is kind of quirky and this is the first time I am noticing it, ```exportConfig``` (passed as an input variable for the modal function) does not get passed on to ```$scope.export``` function by default. It has to be added to the closure to be available to subsequent functions inside the closure. Didn't know JS had this kind of closure problems :smile:

**Comment by elfenheart** [comment]
:+1: LGTM

**Comment by elfenheart** [comment]
LGTM

**Pull request merged**